fx: Sprint #1 4, 5

Bubnov-Roma opened the pull request:
fix: update configs

Commit:
test: check husky and prettier work

Commit:
fix: update husky set up

Commit:
feat: add BEM rules to stylelint

Commit:
test: testing work pre-commit

Commit:
fix: refactor tsconfig, remove excess

Commit:
fix: refactor linstagedrc

Commit:
fix: refactor stylelint.rc , remove BEM pattern rule

Commit:
fix: refactor eslint config, add warn no-console, fix sourceType

Commit:
fix: refactor jest config (remove comment)

Commit:
fix: refactor pre-commit (add 'stylelint' and 'format' rules)

Commit:
fix: refactor prettierrc (add 'endOfLine': 'lf')

Commit:
fix: refactor package.json (add 'init' command, fix path, update version)

SergeySkakun committed:
fix: resolve merge conflict into 'develop' branch

Commit:
test: check pre-commit stylelint rule

SergeySkakun reviewed (changes requested):
Все работает. Парочка маленьких моментов осталось только. В остальном отлично.
Я исправил merge conflict c файлами из `./src`, добавил нужные пункты в README.md насчет скрипта `init`, так что стоит сначала сделать `git pull`, чтобы подтянуть мои последние коммиты, перед тем, как отправишь свой с финальными изменениями.

SergeySkakun committed:
docs: update README.md

change Setup 2. Install dependencies
add "init" script in Available Scripts table

Commit:
chore: update branch

Commit:
fix: remove double rules

Commit:
fix: remove double import

SergeySkakun reviewed (approved):
Отлично справился со всеми проблемами! 💪
